Formula/ncmpcpp.rb

```diff
@@ -31,6 +31,7 @@ class Ncmpcpp < Formula
   depends_on "pkg-config" => :build
   depends_on "libmpdclient"
   depends_on "readline"
+  depends_on "ncurses"
```

Comment:
It would not build without, stating it required ncursesw, which is the devel libraries that I assume are not included in macOS